admin/superuser-init.py

```diff
@@ -1,290 +1,139 @@
 #!/usr/bin/python
 import sys, os, shutil
 import json
+import re
 import argparse
 import getpass
 import openreview
-import utils
 import datetime
 
 """
-REQUIRED ARGUMENTS
+This script should be used by OpenReview administrators to automatically build
+the basic infrastructure for a new conference. It will create (or fill in as
+needed) a directory under openreview-scripts/venues with the necessary
+subdirectories, and will generate the scripts, webfields, and process functions
+needed to run a simple conference.
 
+REQUIRED KEYWORD ARGUMENTS
     conf - the full path of the conference group you would like to create.
         e.g. auai.org/UAI/2017
+        A directory with path exactly equal to the conference group ID
+        should already exist before running this script.
 
-OPTIONAL ARGUMENTS
+    data - a file containing parameters of the conference.
+        See openreview-scripts/admin/conference-template/params.data
 
+OPTIONAL KEYWORD ARGUMENTS
     overwrite - if present, overwrites the conference directory.
-    data - a .json file containing parameters that are otherwise prompted for in the script
     baseurl -  the URL of the OpenReview server to connect to (live site:
         https://openreview.net)
     username - the email address of the logging in user
     password - the user's password
 
-Gathers information to fill in config.py and the webfield/conf.html.
-Creates directories and groups for the entire CONF directory structure.
-Creates sub-directores python/, process/, data/ and webfield/.
-Creates config.py, admin-init.py, submissionProcess.js, commentProcess.js and conf.html.
-Optionally creates admin user profile
+
 """
 
+def parse_json(file):
```

Comment:
Why don't use this to load a json?

```
import json

with open('strings.json') as json_data:
    d = json.load(json_data)
    print(d)
```

Reply:
I said this above but the reason for this is that the json format doesn't accept comments, and I want comments in the params file